openwisp_network_topology/visualizer/views.py

```diff
@@ -26,6 +34,17 @@ class TopologyDetailView(View, GraphVisualizerUrls):
 
     def get(self, request, pk):
         topology = get_object_or_404(self.topology_model, pk)
+        user = request.user
+        if TOPOLOGY_API_AUTH_REQUIRED and not user.is_superuser:
+            auth_perm = check_auth_perm(request)
+            if auth_perm is not None:
+                return auth_perm
+            if str(topology.organization.pk) not in user.organizations_managed:
```

Comment:
please use the [is_manager helper](https://github.com/openwisp/openwisp-users/#is-manager-org)

Reply:
Remember this one please

Reply:
done.